### PR TITLE
Ocd 3435 - Clean up duplicate ONC and ONC-ACB scheduled jobs report

### PR DESCRIPTION
**Pull request by tmy1313**
OCD-3435

### CODE REVIEW

File: chpl/chpl-resources/src/main/resources-production/user-triggers.xml [hunk 1]
```diff
@@ -4,12 +4,7 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://www.quartz-scheduler.org/xml/JobSchedulingData http://www.quartz-scheduler.org/xml/job_scheduling_data_2_0.xsd"
     version="2.0">
-    <pre-processing-commands>
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources-production/user-triggers.xml:7]: These pre-processing commands were here for a reason. I don't want to remove them unless we're sure they can be

File: chpl/chpl-resources/src/main/resources/email.properties [hunk 2]
```diff
@@ -22,7 +22,6 @@ questionableActivityNoDataEmailBody=<p>No questionable activity was found betwee
 #pending change request email properties
 pendingChangeRequestReportFilename=pending-change-request
 pendingChangeRequestEmailSubject=Pending Change Request Report
-pendingChangeRequestEmailAcbSubject=ONC-ACB Pending Change Request Report
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources/email.properties:25]: There were four duplicate reports; did only one of them have values in this property file?

File: chpl/chpl-resources/src/main/resources/jobs.xml [hunk 3]
```diff
@@ -186,7 +126,7 @@
                 </entry>
                 <entry>
                     <key>email</key>
-                    <value></value>
+                    <value>amore@ainq.com☺aiyer@ainq.com</value>
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources/jobs.xml]: Will this overwrite whatever value exists in PROD? We might need to treat this job separately if so. Even if not, probably don't need that second email address in here

File: chpl/chpl-resources/src/main/resources/jobs.xml [hunk 4]
```diff
@@ -597,12 +321,48 @@
...
+            <durability>true</durability>
+            <recover>false</recover>
+            <job-data-map>
+                <entry>
+                    <key>authorities</key>
+                    <value>ROLE_ADMIN;ROLE_ONC</value>
+                </entry>
+            </job-data-map>
+        </job>
+
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources/jobs.xml:364]: Can we add a comment above this job that marks the transition to ONC-ACB specific jobs? Something like the ` <!-- Real World Testing Eligibility Job -->` things above?

**Review by andlar — CHANGES_REQUESTED** [review]

File: chpl/chpl-resources/src/main/resources-production/user-triggers.xml (hunk 1 above)
Comment (tmy1313) [reply chpl/chpl-resources/src/main/resources-production/user-triggers.xml:7]: The pre-processing commands are still in `resources/jobs.xml` which is used by all environments.

File: chpl/chpl-resources/src/main/resources/email.properties (hunk 2 above)
Comment (tmy1313) [reply chpl/chpl-resources/src/main/resources/email.properties:25]: This was the only report that used a different subject for the two emails.

File: chpl/chpl-resources/src/main/resources/jobs.xml (hunk 3 above)
Comment (tmy1313) [reply chpl/chpl-resources/src/main/resources/jobs.xml]: This has been addressed with `<ignore-duplicates>` & `<overwrite-existing-data>` changes in jobs.xml

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by kekey1 — APPROVED** [review]

**Review by andlar — APPROVED** [review]

**Pull request merged**